test/Feature/AndOrParamsSpec.hs

```diff
@@ -80,6 +80,19 @@ spec =
               {"text_search_vector": "'amus':5 'fair':7 'impossibl':9 'peu':4" },
               {"text_search_vector": "'art':4 'spass':5 'unmog':7"}
             ]|] { matchHeaders = [matchContentTypeJson] }
+          get "/tsearch?or=(text_search_vector.plfts(german).Art,text_search_vector.plfts(french).amusant,text_search_vector.not.wfts(english).impossible)"
+            `shouldRespondWith` (
+                if actualPgVersion >= pgVersion112 then
+                    [json|[
+                        {"text_search_vector": "'also':2 'fun':3 'possibl':8" },
+                        {"text_search_vector": "'ate':3 'cat':2 'fat':1 'rat':4" },
+                        {"text_search_vector": "'amus':5 'fair':7 'impossibl':9 'peu':4" },
+                        {"text_search_vector": "'art':4 'spass':5 'unmog':7" }
+                    ]|] { matchHeaders = [matchContentTypeJson] }
+                else
```

Comment:
You can use [when](https://hackage.haskell.org/package/base-4.12.0.0/docs/Control-Monad.html#v:when) instead of having to check the `else` case for version of pg  that don't support `websearch_to_tsquery`.

That would simplify all of the new tests.

Reply:
An example:

https://github.com/PostgREST/postgrest/blob/64dc6ab9acc8cf8242697e817cf6dcf22a41026f/test/Feature/RpcSpec.hs#L259-L265